lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -445,7 +445,11 @@ void ProxyRenderDelegate::_InitRenderDelegate(MSubSceneContainer& container) {
                 globalSelection->addObserver(_observer);
             }
 
-            Ufe::Scene::instance().addObjectAddObserver(_observer);
+            #if UFE_PREVIEW_VERSION_NUM >= 2021
+                Ufe::Scene::instance().addObserver(_observer);
+            #else
+                Ufe::Scene::instance().addObjectAddObserver(_observer);
+            #endif
```

Comment:
I guess its minor and a matter of style, but why are the #if/else/endif indented? That makes the other two lines indented. So when we remove this ufe preview check you have to unindent the code back.